src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -445,7 +458,7 @@ async function updateViewForReporting(props: ResourcePropertiesType, oldProps: R
       ...SQL_TEMPLATE_PARAMETER,
     };
 
-    const sqlStatements2 = getUpdatableSql(props.reportingViewsDef, appUpdateProps.oldViewSqls, mustacheParam, '/opt/dashboard');
+    const sqlStatements2 = getUpdatableSql(props.reportingViewsDef, appUpdateProps.oldViewSqls, appUpdateProps.oldViewSqls, mustacheParam, '/opt/dashboard');
```

Comment:
Why pass the same `oldViewSqls` twice?

Reply:
The `getUpdatableSql` already has two parameters the new SQLDefs and old SQLDefs, it's enough to find the newly added SQL statements.

Reply:
typo, first param should be appUpdateProps.oldSchemaSqlArray